ml/Config.cc

```diff
@@ -47,6 +47,9 @@ void Config::readMLConfig(void) {
     double ADWindowRateThreshold = config_get_float(ConfigSectionML, "window minimum anomaly rate", 0.25);
     double ADDimensionRateThreshold = config_get_float(ConfigSectionML, "anomaly event min dimension rate threshold", 0.05);
 
+    double MaxAnomalyRateInfoTableRows = config_get_number(ConfigSectionML, "the maximum size, in rows, of the data table that holds anomaly rate information", 10000.0);
+    double MaxAnomalyRateInfoAge = config_get_number(ConfigSectionML, "the oldest age, in hours, of the anomaly rate information allowed to stay in the table", 240.0);
```

Comment:
These are currently unused. Is this intentional?

Reply:
Not anymore! Removed.